avocado/plugins/testlogs.py

```diff
@@ -63,6 +76,13 @@ def post(self, job):
                 except (FileNotFoundError, PermissionError) as error:
                     LOG_UI.error('Failure to access log file "%s": %s',
                                  path, error)
+        if summary_tests:
+            LOG_UI.info("")
+            LOG_UI.info(" Summary Output:")
+            LOG_UI.info(" (Defined by 'job.output.testlogs.summary_statuses')")
```

Comment:
This feels like it's replacing the documentation, so IMO the message should be dropped and documentation should be added instead which I don't see in this patch.

Reply:
Documentation is auto generated and it will be available on `avocado config reference` and also in our rtd page. But I'm ok removing the message.